src/Chirp.Web/Pages/UserTimeline.cshtml.cs

```diff
@@ -32,8 +34,22 @@ public async Task<IActionResult> OnGet(string author, [FromQuery] int page)
 
         CurrentAuthor = author;
         PageNumber = page;
-        Cheeps = await _service.GetCheepsFromAuthor(author, page);
+        
+        if (User.Identity != null && User.Identity.Name == author) 
```

Comment:
Nice way of ensuring that the logged-in user's messages aren't displayed when visiting others timelines.